pom.xml

```diff
@@ -643,6 +646,8 @@
               <exclude>src/ports/postgres/modules/utilities/admin.py_in</exclude>
               <exclude>src/ports/postgres/modules/utilities/control.py_in</exclude>
               <exclude>src/ports/postgres/modules/utilities/control_composite.py_in</exclude>
+              <exclude>src/ports/postgres/modules/utilities/create_indicators.py_in</exclude>
+              <exclude>src/ports/postgres/modules/utilities/create_indicators.sql_in</exclude>
               <exclude>src/ports/postgres/modules/utilities/data_preparation.py_in</exclude>
```

Comment:
Since we renamed data_preparation to create_indicators, we should remove lines 651 and 652.